bench/workgen/workgen_time.h

```diff
@@ -199,3 +199,9 @@ ts_us(const timespec &ts)
 {
     return (ns_to_us(ts.tv_nsec) + sec_to_us(ts.tv_sec));
 }
+
+inline uint64_t
+secs_us(double secs)
+{
+    return (ns_to_us(secs * NSEC_PER_SEC));
```

Comment:
Why multiply to nanosecs and then divide to usecs? Why not just use `secs * USEC_PER_SEC` from `workgen_time.h`?

Reply:
Right!  fixed.